start/scaling-up/preparing-to-scale-up.md

```diff
@@ -132,12 +152,35 @@ desired output, and do not conflict with each other when submitted together. Onc
 you are confident that the jobs will complete as desired, then scale up to submitting 
 the entire set of jobs.
 
-## Avoid Exceeding Disk Quotas
+# Monitoring Job Status and Obtaining Run Information
+
+Gathering information about how, what, and where a job ran can be important for both troubleshooting and optimizing a workflow. The following commands are a great way to learn more about your jobs: 
+
+| Command      | Description |
+| ----------- | ----------- |
+| `condor_q`      | Shows the queue information for your jobs. Includes information such as batch name and total jobs. |
+| `condor_q <JobID> -l` | Prints all information related to a job including attributes and run information about a job in the queue. Output includes `JobDurationCategory`, `ServerTime`, `SubmitFile`, etc. |

```

Comment:
```suggestion
| `condor_q <JobID> -l` | Prints all information related to a job including attributes and run information about a job in the queue. Output includes `JobDurationCategory`, `ServerTime`, `SubmitFile`, etc. |
```suggestion
| `condor_q <JobID> -l` | Prints all information related to a job including attributes and run information about a job in the queue. Output includes `JobDurationCategory`, `ServerTime`, `SubmitFile`, etc. Also works with `condor_history`. |

```
```